### PR TITLE
Leveraging Open Source as a Strategy

### PR DESCRIPTION
**Pull request by gyehuda**
Related to Issue #144 
consider this text as a first draft.

### CODE REVIEW

**Review by psmulovics — APPROVED** [review]

**Comment by psmulovics** [comment]
@netlify rebuild

File: docs/bok/Activities/Level-5/Leveraging-Strategically.md [hunk 1]
```diff
@@ -5,19 +5,32 @@ tags:
   - Data Leakage (Risk)
   - Placeholder
   - Level 5 (OSMM)
-draft: true
 ---
 
-Refer to [Strategy](../Artifacts/Strategy) as the _output_ from this activity.
+#Level 5 is about [Strategy](../Artifacts/Strategy)
```
Comment (thomascooper-rbc) [inline docs/bok/Activities/Level-5/Leveraging-Strategically.md:10]: Making this a header.
```suggestion
# Level 5 is about [Strategy](../Artifacts/Strategy)
```

**Comment by robmoffat** [comment]
How's this coming along, @gyehuda ?

**Comment by robmoffat** [comment]
Hi Gil,

Any progress on this?  It would be really nice to have a draft of this for review in the 16th August SIG meeting, if you could make it.

**Comment by gyehuda** [comment]
I've updated the document. I think it's ready for review and about as complete as I think I can make it. I've ensured the voice is consistent. Sorry for the delay in getting this done. It's unclear if I'll be able to join the Aug 16th meeting -- depends on some potential travel conflicts.

**Comment by robmoffat** [comment]
@gyehuda here you go: https://deploy-preview-145--open-source-readiness.netlify.app/docs/bok/Activities/Level-5/Leveraging-Strategically

**Comment by robmoffat** [comment]
@gyehuda are you happy with this?  We're ready to merge now...

**Comment by gyehuda** [comment]
I am, thanks. I think it's a good first version and we can always update this as the other content comes together. I'm glad to contribute a small part of this important effort.

**Pull request merged**